.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: java
+jdk:
+  - openjdk11
```

Comment:
czemu i nie na 14? :)
ostatnio modna jest akcja #AllTestsGreenOnJDK14 :)

Reply:
JDK 14 zostało opublikowane mniej więcej 2 tygodnie temu, nie zdążyłam się jeszcze z nim zapoznać. Dodatkowo, ta wersja nie jest LTS i nawet jak się już zapoznam, to używać będę tylko we własnej piaskownicy.